doc/analysis/index.rst

```diff
@@ -61,3 +61,8 @@ JSON specification in ``model.json``.
     >>> dm = analysis[0].get_design_matrix(subject='01', run=1, task='taskA')
     >>> # Sample query: retrieve session-level contrast matrix
     >>> cm = analysis[1].get_contrasts(subject='01', session='retest')
+
+
+.. note::
+
+    For a more detailed set of examples, please refer to the `tutorial <https://github.com/bids-standard/pybids/blob/0.14.0/examples/statsmodels_tutorial.ipynb>`_
```

Comment:
Do we want to use `master` instead of `0.14.0`?

Reply:
It seems that the documentation is updated every time a tag is pushed, so I can foresee a situation where the code in the notebook is updated to a future release and doesn't work with the current release. That's why I pinned this link to the current release tag. I wonder whether there is a URL that always points to the most recent tagged version of a repo.

Reply:
I don't think so...